### PR TITLE
Updated seed_test script with new structure

### PR DESCRIPTION
**Pull request by radtriste**
and simplified GHA composite action

I just extracted what was in the composite action to the shell script, so we can reuse it locally.

### CODE REVIEW

File: .github/workflows/jenkins-tests.yml [hunk 1]
```diff
@@ -42,7 +42,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:    
     - name: DSL tests
-      uses: kiegroup/kogito-pipelines/.ci/actions/dsl-tests@main
+      # uses: kiegroup/kogito-pipelines/.ci/actions/dsl-tests@main
+      uses: radtriste/kogito-pipelines/.ci/actions/dsl-tests@update_seed_test_script # test to remove
```
Comment (radtriste) [inline .github/workflows/jenkins-tests.yml]: gonna set back before the merge

**Review by Ginxo — APPROVED** [review]
LGTM, just use `kiegroup` at `.github/workflows/jenkins-tests.yml#L46` before merging as you mentioned

File: .ci/actions/dsl-tests/action.yml [hunk 2]
```diff
@@ -30,8 +30,10 @@ runs:
   - name: Checkout Seed repo
     uses: actions/checkout@v2
     with:
-      repository: kiegroup/kogito-pipelines
-      ref: main
+      # repository: kiegroup/kogito-pipelines # test to remove
+      # ref: main
+      repository: ${{ github.event.pull_request.head.repo.full_name }}
```
Comment (radtriste) [inline .ci/actions/dsl-tests/action.yml]: gonna set back before the merge

**Comment by radtriste** [comment]
all green. removing the test commit

archive: https://github.com/kiegroup/kogito-pipelines/runs/7107402097?check_suite_focus=true

**Pull request merged**